src/test/java/kitchenpos/products/tobe/domain/DisplayedNameTest.java

```diff
@@ -0,0 +1,28 @@
+package kitchenpos.products.tobe.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class DisplayedNameTest {
+
+    @DisplayName("`DisplayedName`에는 `name`이 필수다")
```

Comment:
테스트 코드에 요구사항을 그대로 사용하는 것도 좋은 방법이 되겠지만 
제가 테스트 코드 네이밍을 하는 방식도 공유해드릴게요. 참고만 부탁드려요 ! 
```suggestion
    @DisplayName("`DisplayedName`생성시 `name`이 존재 하지 않으면 IllegalArgumentException을 던진다.")
```